mocca-client/src/main/java/com/paypal/mocca/client/MoccaSerializer.java

```diff
@@ -272,6 +258,40 @@ private void writeRequestPojo(final ByteArrayOutputStream outputStream, final Ty
         }
     }
 
+    private String objectToString(final Object v, String key, final List<String> ignoreFields) {
+        if (v instanceof String) {
+            return "\\\"" + v + "\\\"";
+        } else if (v instanceof Number || v instanceof Boolean) {
+            return String.valueOf(v);
+        } else if (v instanceof List) {
+            List<?> listElement = (List)v;
```

Comment:
```suggestion
        } else if (v instanceof Collection) {
            Collection<?> listElement = (Collection)v;
```

With this, `Set` worked in  your sample test.  Do you think there is harm in being broader than `List`?

Reply:
it should be ok, I think. @fabiocarvalho777 ?

Reply:
My only thought is that when the request is deserialized it would almost certainly be coerced into a List, which would imply ordering for the original collection that did not exist. This is really minor though and if it is useful to change to Collection from List I can do so easily. I'll leave it up to @fabiocarvalho777 to decide.

Reply:
Yes, GraphQL doesn't support any collections but list. It doesn't supports maps or sets. We also mention `List` necessarily in our end user documentation. So it is better to leave it as `List` please.

Reply:
ok